lib/ex_unit/lib/ex_unit.ex

```diff
@@ -180,6 +180,9 @@ defmodule ExUnit do
       and print them on test failure. Can be overridden for individual tests via
       `@tag capture_log: false`. Defaults to `false`.
 
+    * `:case_load_timeout` - the timeout to be used when loading a test case.
+      Defaults to 60_000ms
```

Comment:
Also, maybe we can format this in a nicer way by using backticks:

```
Defaults to `60_000` milliseconds.
```

I think it reads a bit cleaner :)